it.unibo.tuprolog.ide.yml

```diff
@@ -0,0 +1,37 @@
+id: it.unibo.tuprolog.ide
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - "org.freedesktop.Sdk.Extension.openjdk17"
+command: tuprolog-ide
+finish-args:
+  - "--env=PATH=/app/jre/bin:/app/bin:/usr/bin"
+  - "--env=JAVA_HOME=/app/jre"
+  - "--share=ipc"
+  - "--socket=x11"
+  - "--device=dri"
+  - "--filesystem=home"
```

Comment:
who does it need home ?

Reply:
I tried without home and I was not able to save files outside the container. Maybe I'm missing something here. Any way to do it better?

Reply:
i guess it doesn't support the filechooser portal.

Reply:
I don't think it does, unfortunately. I could limit to a specific folder like Documents though, but I don't know